sys/include/ztimer.h

```diff
@@ -301,13 +301,26 @@ struct ztimer_base {
     uint32_t offset;            /**< offset from last timer in list */
 };
 
+/**
+ * @defgroup    ztimer_now64 ztimer_now64
+ * @deprecated  use uint32_t or ztimer64, will be removed after 2022.07 release.
```

Comment:
From just reading this (and below) this is a bit confusing: One is a type, one is a module. The detailed text added below is much more helpful. So rather add it here? Also, the deprecation cycle usually is 2 releases:

```suggestion
 * @deprecated use @ref ztimer_now() returning uint32_t or alternatively use module
 *             `ztimer64` with @ref ztimer64_now() returning uint64_t.
 *             Will be removed after 2022.10 release.
```